lib/sql.js

```diff
@@ -601,8 +623,33 @@ SQLConnector.prototype.updateAttributes = function(model, id, data, options, cb)
  * @param {Function} cb The callback function
  * @returns {ParameterizedSQL} The UPDATE SQL statement
  */
-SQLConnector.prototype.buildUpdate = function(model, where, data, options) {
+SQLConnector.prototype._buildUpdate = function(model, where, data, options) {
   var fields = this.buildFieldsForUpdate(model, data);
+  return this._constructUpdateQuery(model, where, fields);
+};
+
+/**
+ * Build the UPDATE statement for replacing
+ * @param {String} model The model name
+ * @param {Object} where The where object
+ * @param {Object} data The data to be changed
+ * @param {Object} options The options object
+ * @param {Function} cb The callback function
+ * @returns {ParameterizedSQL} The UPDATE SQL statement for replacing fields
+ */
+SQLConnector.prototype._buildReplace = function(model, where, data, options) {
```

Comment:
Hi @raymondfeng,

Sorry still this comment is confusing to me... `_buildReplace` exactly has the same signature of  `buildUpdate`; please see line 626 in this PR or [here](https://github.com/strongloop/loopback-connector/blob/master/lib/sql.js#L604) in the original code. Also, it is worth mentioning `_buildReplace` is a new function, which I just added; so does it need to have a different function signature of `buildUpdate`? They both have similar functionality and I already used the same signatures for both of them in terms of consistency...

My other question: is based on [this comment](https://github.com/strongloop/loopback-connector/pull/35#discussion_r49397747) I reverted back `_buildUpdate ` to `buildUpdate` in terms of consistency would you like me to change `_buildReplace ` to `buildReplace ` as well?

Thanks!